plugins/fishlim/plugin_hexchat.c

```diff
@@ -138,7 +138,7 @@ static int handle_outgoing(char *word[], char *word_eol[], void *userdata) {
 static int handle_incoming(char *word[], char *word_eol[], hexchat_event_attrs *attrs, void *userdata) {
     const char *prefix;
     const char *command;
-    const char *recipient;
+    const char *channel = hexchat_get_info(ph, "channel");
```

Comment:
Wow, I think my fishlim conf was messed up with self keys, they were being used instead of the senders. I removed the self keys and removed the changes to this file and it works.